From dcafc3a3ef62b185e6d96ed32f6712c065ee5e51 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Thu, 2 Apr 2020 18:36:36 +0800
Subject: [PATCH 19/22] driver: media: mxc-mipi-csi2_yav: initialize bus_type
 member of structure v4l2_fwnode_endpoint

It is need to initialize bus_type member of structure
v4l2_fwnode_endpoint. Otherwise, it is a random value,
and maybe causes trace call when it is used in later.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/imx8/mxc-mipi-csi2_yav.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/imx8/mxc-mipi-csi2_yav.c b/drivers/media/platform/imx8/mxc-mipi-csi2_yav.c
index 87b17160ad80..53b16f2d13a8 100644
--- a/drivers/media/platform/imx8/mxc-mipi-csi2_yav.c
+++ b/drivers/media/platform/imx8/mxc-mipi-csi2_yav.c
@@ -463,7 +463,7 @@ static int mipi_csi2_parse_dt(struct mxc_mipi_csi2_dev *csi2dev)
 {
 	struct device *dev = &csi2dev->pdev->dev;
 	struct device_node *node = dev->of_node;
-	struct v4l2_fwnode_endpoint endpoint;
+	struct v4l2_fwnode_endpoint endpoint = { .bus_type = 0 };
 	u32 i;
 
 	csi2dev->id = of_alias_get_id(node, "csi");
-- 
2.17.1

